lib/resolve-script/glob-patterns.js

```diff
@@ -1,14 +1,31 @@
 var path = require('path')
+var resolvePkg = require('resolve-pkg')
 
-module.exports = function (dir1, dir2) {
+module.exports = function (dir1, dir2, options) {
   var expanded = path.resolve(dir1, path.join.apply(this, dir2.split(':')))
 
-  return [
+  var globArray = [
     // exact file match (+ any same-named extensions)
     expanded + '+(|.*)',
     // a nested index file match (+ any same-named extensions)
     path.join(expanded, 'index+(|.*)'),
     // any nested files at all
     path.join(expanded, '*')
   ]
+
+  if (options && Array.isArray(options.modules)) {
+    options.modules.forEach(function (moduleName) {
+      const modulePath = resolvePkg(moduleName + '/scripts')
```

Comment:
I should be more clear:

1. Windows command support is implemented in a custom way in Scripty. If you're on Windows, it will first look in a mirrored directory hierarchy under `scripts-win/`, not `scripts/`. That means for you to keep Windows support working consistently, this feature would need to resolve windows paths in the same way other scripts are loaded (via this [bit of code here](https://github.com/testdouble/scripty/blob/a003adc28b39c57bffa394714d5da6e2bca69d06/lib/resolve-script/script-dirs.js#L13-L17)). 

2. Whoops, fixed the Appveyor permission error. You can see a [build log here](https://ci.appveyor.com/project/searls/scripty/build/job/1k6wkfw5a4owk8ff). What I think this PR really needs to be sure it works is a "safe" (i.e. integration) test that uses a new set of minimal nested module fixtures to verify that script-module loading works as you expect it to, and that also exercises scripts-win. I think if you look under the `safe/` path you should find readily imitatible tests to start from